packages/README.md

```diff
@@ -32,10 +33,39 @@ When creating a new package, you need to provide at least the following:
     	},
     	"publishConfig": {
     		"access": "public"
-    	}
+    	},
+    	// Include this line to include the package as a WordPress script.
+    	"wpScript": true,
+    	// Include this line to include the package as a WordPress script module.
+    	"wpScriptModuleExports": "./build-module/index.js"
```

Comment:
Can I ask why did we add these properties and what happens if we don't provide them?